simplified-books-time-tracking/README.md

```diff
@@ -0,0 +1,5 @@
+org.librarysimplified.books.time.tracking
+===
+
+The `org.librarysimplified.book.time.tracking` module provides 
```

Comment:
Provides what?! Provides what?!?!

Reply:
I was just trying to keep things interesting and mysterious :D